CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - [#8446](https://github.com/MetaMask/metamask-extension/pull/8446): Fix popup not opening
 - [#8449](https://github.com/MetaMask/metamask-extension/pull/8449): Skip adding history entry for empty txMeta diffs
 - [#8447](https://github.com/MetaMask/metamask-extension/pull/8447): Delete Dai/Sai migration notification
+- [#8454](https://github.com/MetaMask/metamask-extension/pull/8454): Clean up list of available currencies
```

Comment:
Oh, actually... maybe this shouldn't have an entry 🤔 It isn't a user-facing change. I'll remove this.